docs/about-this-documentation.rst

```diff
@@ -0,0 +1,72 @@
+.. _about-this-documentation:
+
+About this documentation
+========================
+
+The documentation is Rockcraft's first-person account of itself to the world. It
+provides knowledge and guidance to people who use, study, and develop the project.
+
+Rockcraft is built by many people, and its documentation voices the intent of its
+developers and the practical needs of its users. As Rockcraft changes, so will people's
+understanding and experience of it. Its documentation must therefore continuously evolve
+in simpatico.
+
+Canonical is intent on advancing the art and science of documentation, and Rockcraft is
+a space for such innovation. This documentation is an application of ideas such as
+`technical authoring
+<https://ubuntu.com/blog/documentation-development-and-design-for-technical-authors>`_,
+`Diátaxis <https://diataxis.fr>`_, and learning through self-driven practice.
+
+
+Contributions
+-------------
+
+Because Rockcraft is open source and made possible by its community, it accommodates
+insights into its operation and documentation from everyone. That includes you!
+
+The Rockcraft team welcomes changes, suggestions, and questions. If you want to help
+author the documentation, there are a few paths you can take:
```

Comment:
FYI there is a forum for rockcraft: https://discourse.ubuntu.com/c/project/rockcraft/118

Reply:
As it's more of a tag in the Ubuntu Discourse instance than a forum, I didn't feel like including it. Do we want to promote it?

Reply:
Let's leave it as-is for now. :+1: